scripts/wl-present

```diff
@@ -17,7 +17,7 @@ usage() {
     echo "  fullscreen                  fullscreen the wl-mirror window"
     echo "  no-fullscreen               unfullscreen the wl-mirror window"
     echo "  fullscreen-output [output]  set the output to fullscreen to (default asks via slurp)"
-    echo "  no-fullscreen-output        set the output to fullscreen to to the current output"
+    echo "  no-fullscreen-output        set the output to fullscreen to the current output"
```

Comment:
This wasn't actually a typo, but you're right that the previous version doesn't read very correct. Something still bugs me about the new version though, but I'm struggling to find a better one.